packages/biocaml/biocaml.0.11.2/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["philippe.veber@gmail.com"]
+authors: [
+  "Sebastien Mondet"
+  "Ashish Agarwal"
+  "Philippe Veber"
+  "Christophe Troestler"
+  "Sergei Lebedev"
+  "David Koppstein"
+  "Alexander Gryzlov"
+  "Francois Berenger"
+]
+bug-reports: "https://github.com/biocaml/biocaml/issues"
+homepage: "https://github.com/biocaml/biocaml/"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+dev-repo: "git+https://github.com/biocaml/biocaml.git"
+synopsis: "The OCaml Bioinformatics Library"
+description: """
+
+Biocaml aims to be a high-performance user-friendly library for
+Bioinformatics. See the [website](http://biocaml.org) for detailed API
+documentation. We welcome contributors and feedback from users. Please
+contact us with any comments and suggestions for features you would
+like added.
+"""
+depends: [
+  "base64"
+  "dune" {>= "1.10"}
+  "core" {>= "v0.15.0"}
```

Comment:
Looks like moving to 0.15 breaks phylogenetics:
```
+ /home/opam/.opam/opam-init/hooks/sandbox.sh "build" "dune" "build" "-p" "phylogenetics" "-j" "31" "@install" (CWD=/home/opam/.opam/4.14/.opam-switch/build/phylogenetics.0.1.0)
- (cd _build/default/lib && /home/opam/.opam/4.14/bin/menhir newick_parser.mly --compile-errors newick_parser.messages) > _build/default/lib/newick_parser_errors.ml
- Read 3 sample input sentences and 2 error messages.
- File "lib/dune", line 6, characters 13-88:
- 6 |  (preprocess (pps ppx_deriving.show ppx_inline_test ppx_jane -allow-unannotated-ignores)))
-                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
- (cd _build/default && .ppx/225a9d522116ae1bff0c29e21b1aa610/ppx.exe -allow-unannotated-ignores --cookie 'inline_tests="disabled"' --cookie 'library-name="phylogenetics"' -o lib/newick_parser__mock.ml.pp.mock --impl lib/newick_parser__mock.ml.mock -corrected-suffix .ppx-corrected -diff-cmd - -dump-ast)
- .ppx/225a9d522116ae1bff0c29e21b1aa610/ppx.exe: unknown option '-allow-unannotated-ignores'.
- ppx.exe [extra_args] [<files>]
-   -as-ppx                          Run as a -ppx rewriter (must be the first argument)
-   --as-ppx                         Same as -as-ppx
-   -as-pp                           Shorthand for: -dump-ast -embed-errors
-   --as-pp                          Same as -as-pp
-   -o <filename>                    Output file (use '-' for stdout)
```

You probably need to just add a conflict with ppx_jane > 0.15.0 there.

Reply:
Thanks Marcello, this is fixed upstream, and I intended to submit a new release for `phylogenetics` as soon as this release of `biocaml` is published.

Reply:
This is fixed in a separate PR